server/game/cards/02-AoA/MightMakesRight.js

```diff
@@ -13,6 +15,7 @@ class MightMakesRight extends Card {
                 gameAction: ability.actions.sacrifice()
             },
             then: {
+                condition: context => context.preThenEvents && context.preThenEvents.filter(event => !event.cancelled).reduce((total, event) => total + event.card.power, 0) > 25 ? 1 : 0,
```

Comment:
I think the right way to do this is "may: 'Forge a key'", see chota for an example

Reply:
As per our discussion - MMR does not allow for optional forging. If you choose to sacrifice, it MUST be creatures with a total Power of >= 25, and if you sacrifice, you MUST forge.